recipes/qt-wayland/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "5.15.8" %}
+package:
+  name: qt-wayland
+  version: {{ version }}
+
+source:
+  url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/qtwayland-everywhere-opensource-src-{{ version }}.tar.xz
+  sha256: fa0e7bd40f5da2d30fbdb51a436c5776950d6bd070b61a8d62d85cb537934529
+
+build:
+  skip: true   # [not linux]
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt-wayland', max_pin='x.x') }}
+
+requirements:
+  build:
+    - perl
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - qt-main                            # [build_platform != target_platform]
+    - sysroot_linux-64 2.17              # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl') }}           # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+  host:
+    - qt-main  {{ version }}
+    - libxkbcommon
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-libxrandr
+    - xorg-libxcomposite
+    - wayland
+    - fontconfig
+    - freetype
+    - zlib
+    - glib
+  run:
+    # https://github.com/conda-forge/xorg-libx11-feedstock/pull/28
```

Comment:
Is it necessary to wait for that PR before merging this one?

Reply:
Not, it just refers to the run export, and why we need to specify it here. when that one is closed, we can likely remove the dependency